scriptbuilder/pom.xml

```diff
@@ -83,6 +83,6 @@
         </configuration>
       </plugin>
     </plugins>
-  </build>
+  </build>-->
```

Comment:
I added the guava version range override in `jclouds-project`, but `jclouds-scriptbuilder` didn't get them because it had these settings for `maven-bundle-plugin`. It appears to me that these are unnecessary, since they are exactly the same as in `jclouds-project` - maybe they were left here by accident?

At first I commented them out, but I think I'll simply remove them when updating the PR.